<dec f='codebrowser/external/libcxxabi/include/cxxabi.h' l='38' type='void * __cxa_allocate_exception(size_t thrown_size)'/>
<doc f='codebrowser/external/libcxxabi/include/cxxabi.h' l='37'>// 2.4.2 Allocating the Exception Object</doc>
<def f='codebrowser/external/libcxxabi/src/cxa_exception.cpp' l='159' ll='166' type='void * __cxa_allocate_exception(size_t thrown_size)'/>
<doc f='codebrowser/external/libcxxabi/src/cxa_exception.cpp' l='154'>//  Allocate a __cxa_exception object, and zero-fill it.
//  Reserve &quot;thrown_size&quot; bytes on the end for the user&apos;s exception
//  object. Zero-fill the object. If memory can&apos;t be allocated, call
//  std::terminate. Return a pointer to the memory to be used for the
//  user&apos;s exception object.</doc>
